R/utils.R

```diff
@@ -10,40 +10,37 @@ compact <- function(x) {
   x[!is_empty]
 }
 
-"%||%" <- function(a, b) if (!is.null(a)) a else b
+"%||%" <- function(a, b) if (!is.null(a) && !(length(a) == 1 && is.na(a))) a else b
```

Comment:
I made this change because desc returns `NA` for missing fields rather than `NULL`. I can change the use of `%||%` in the package to the equivalent `if`.